### DISCUSSION TITLE
Add ability to exclude certain types of elements from database backups

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/cms#10446] [Ideas]
Passing around database backups is an integral part of the development process; however, we really shouldn't be passing around database backups with PII embedded in them. Ideally, Craft should give admins the option to broadly omit certain types of content from database backups. Here are some examples:

- Freeform Submissions
- Craft Commerce Orders & Customers
- Non-admin users (I realize this one is more problematic)
- Formie Submissions
- Campaign Contacts

Currently, we can exclude specific tables; however, that won't solve this particular case.

**Comment by timkelty** [comment]
> Currently, we can exclude specific tables; however, that won't solve this particular case.

Which of those _wouldn't_ be covered by excluding tables (except users).
